src/liballoc/boxed.rs

```diff
@@ -816,6 +827,18 @@ impl From<Box<str>> for Box<[u8]> {
     }
 }
 
+#[unstable(feature = "box_into_boxed_slice", issue = "none")]
+impl<T> From<Box<T>> for Box<[T]> {
```

Comment:
AFAIK, trait impls are effectively insta-stable -- they can't really be unstable unless the trait and/or types involved are unstable.

Reply:
ha, so I'll remove this from, and it can be added back if/when this is stable